host/debian/changelog

```diff
@@ -0,0 +1,6 @@
+xap-hss-host (0.1) stable; urgency=low
```

Comment:
The common scheme for out-of-tree module packages in Debian [is](https://packages.debian.org/search?keywords=dkms&searchon=names&suite=buster&section=all) `<module>-dkms`.  So let's call this `hss-dkms` instead of `xap-hss-host`.

Reply:
Done